lib/commands/run.js

```diff
@@ -106,9 +106,16 @@ function run(scriptPath, options) {
             return callback(null, context);
           });
       } else if (context.script.config.payload &&
-                 _.isObject(context.script.config.payload) &&
-                 options.payload) {
-        let csvdata = fs.readFileSync(options.payload, 'utf-8');
+                 (_.isObject(context.script.config.payload) ||
+                 options.payload)) {
```

Comment:
Don't think we need this check (as we're using `options.payload` if it's available)